src/main/java/athena/commands/AddCommand.java

```diff
@@ -35,11 +36,13 @@ public AddCommand(String name, String startTime, String duration, String deadlin
      * @param ui       Ui
      */
     @Override
-    public void execute(TaskList taskList, Ui ui) {
+    public void execute(TaskList taskList, Ui ui) throws AddException {
+        if (taskName.equals("") || taskStartTime.equals("")) {
+            throw new AddException();
```

Comment:
I feel that it would be better to have a more specific exception name.

Reply:
I'll change it to AddMissingRequiredParametersException